### ISSUE TITLE
An Old Hungarian font required with optimized ligatures

### ISSUE THREAD

**Issue by tamasbartos** [googlefonts/noto-source#400]
@marekjez86 I know, that it's not the userside discussion, but whoose asked me, don't speak in English. We discussed about that, the ligatured version need only consonant+vowel and double consonant ligatures.
Because of the philosophy of Noto-fonts developers, that remove glyphs from a font isn't allowed, and using the given ligatured NotoSansOldHungarian font's discrete ligatures make chaos, I'll do fire-fighting kind of repair of the NotoSansOldHungarian font.  I'll start a font project called "NotoSansOldHungarian optimized", which will be follow philosophy, as I wrote above.

So, our target an Old Hungarian font with double consonant, syllable like ligatures formed consonant+vowel and added two special ligatures as "és" (and)  "az" (the or that).

**Comment by tamasbartos** [comment]
@marekjez86 All right, Marek, I close this issue. I'll work on NotoSansOldHungarian font. But I'll remove "ck" ligature, because of that it looks like a penis. This makes ridiculous Old Hungarian writing.

**Comment by tamasbartos** [comment]
@marekjez86 @davelab6 I reopen this issue again. 
I was thinking a lot, I extend the required ligatures with vowel+consonant, which will be simple ligatures only (with ZWJ) , won't be discrete. If somebody want to use, use with ZWJ. 

The consonant+vowel syllable-like ligatures and double consonants can be discrete. This philosophy given from the Hungarian language logic.

I extend it with ligature "q", "w", "y", "x", "dz", "dzs" as non discrete ligatures. If the non-historical ligatures would be discrete, it can be made "tyúxar"-like problems.  
"A kecske is jóllakot, a káposzta is megmaradt." as we Hungarians tell. ( I think about rovasinfo's problem.)

The newest xkeyboard-config's Old Hungarian layout already have ZWJ key on the keypad AltGr+2. There is a layout with "ZWJ" as the top-left position key, called "Old Hungarian (for ligatures)", too. Unfortunately "Short description" of second ones isn't translated correctly to Hungarian language, Gnome users can't choose that.

There is a layout on the keyman.com, which has not ZWJ.

I'll develop a package, wich will have ZWJ.

I would like make a webpage, collecting LO settings, keyman settings, and how can users download NotoSansOldHungarian, NotoSansOldHungarianUI, and my plan "NotoSansOldHungarianOpt" ( optimized ) fonts.
I have any other plan with the webpage.

I'm not willing to include left-to-right glyphs into "my"font.

The Unicode describe strictly about Old Hunarian script, that there is a must to use bi-di algorithm.

NotoSansOldHungarianOpt font will have about 600 small and 600 capital syllable-like ligature. If I would develop light-to-left ligatures it will be duplicate the size of font, unnecessarily.

Making capital ligatures from small ligatures is an easy way, if I use font-designer programs, too. Vowel+consonant ligatures can be mirrored consonant+vowel ligatures.

Several syllable like ligatures already developed in NotoSansOldHungarian font.

**Comment by tamasbartos** [comment]
@davelab6 Can I work on it?

**Comment by tamasbartos** [comment]
@davelab6 @marekjez86 I am sorry, I got a mistake in https://github.com/googlefonts/noto-source/issues/400#issuecomment-900652181
when I wrote "light-to-left". I wanted to write "left-to-right"

**Comment by tamasbartos** [comment]
@davelab6 @marekjez86 I feel like I worked unnecessarily on the "UI" version of the Old Hungarian font.  It has still not been published on the "noto-fonts" project page.

**Comment by tamasbartos** [comment]
@marekjez86 Marek! The developer of this method (using consonant+vowels syllable-like ligatures, double consonants) is András Tisza. He does not recommends the use of vowel + consonant ligatures. He made this system with nearly ten years of experience. I  think I need to adapt to his method.
I read his texts written by this method. It works.
The "összerovás", or ligatures apeared in the history of the Old Hungarian scripts. The target was that, the text could be shorter to fit on the stick they wanted to write. The András Tisza's philosophy is the same: the text could be shorter. 
In summary: the project need to have consonant+vowel and double consonant ligatures, and additionally (it is my opinion) non-historic ligatures, which aren't discrete ligatures, with ZWJ only. I working on it
@davelab6 I'm sorry about the politic comments on noto-fonts project page. I know we have no excuse, but we are Hungarians.
@davelab6  I'm sorry about Tamás Rumi's comments. We also have a lot of trouble with him.

**Comment by tamasbartos** [comment]
@marekjez86  I discussed about using non-historic ligatures whith standardizers. They wrote me that, it is no coincidence that these symbols are not included in the standard.
The following non-historic Old Hungarian letters can be written with historic letters:
Q->K+v
W->V
X->K+sz
Y->I
Dz->D+z
Dzs->D+zs
More than sixty standardizer voted, that non-historic letters are unnecessary and destroy the traditions. Just only three standardizer wanted to include these non-historic letters. Tamás Rumi, László Sípos and Dr. Gábot Hosszú.
I can't decide for myself whether or not to include these non-historical signs as ligatures in the Old Hungarian optimized font.
You can read more articles about the standardization of Old Hungarian script in Hungarian language on the page https://magyarrovas.hu/

**Comment by tamasbartos** [comment]
@marekjez86 In comment https://github.com/googlefonts/noto-source/issues/400#issuecomment-911293428
when I wrote for example "Q->K+v" I were thinkig about these letters' as explanation. "K" means "Old Hungarian K". In the 20th century  appeared "Old Hungarian Q" symbol and the other special symbols, listed in the comment https://github.com/googlefonts/noto-source/issues/400#issuecomment-911293428
which were born as a latinization.

**Comment by tamasbartos** [comment]
@marekjez86 I didn't know that ligatures' development based on Unicode docs. I close it.
